backend/src/controllers/utils/index.ts

```diff
@@ -72,3 +81,186 @@ export const getTemplatesPath = (): string => {
+    const supplierEndpointID = new UdtIdentifier(companyConfig.company.btw, {
+      schemeID: supplierEndpointScheme ? supplierEndpointScheme.schemeID : '9925'
+    });
+
+    const supplierParty = new Party({
+      EndpointID: supplierEndpointID,
+      partyLegalEntities: [supplierLegalEntity],
+      postalAddress: supplierPostalAddress,
+      contact: new Contact({
+        name: companyConfig.company.website,
+        electronicMail: companyConfig.company.email,
+        telephone: companyConfig.company.telephone
+      }),
+      partyTaxSchemes: [new PartyTaxScheme({
+        companyID: companyConfig.company.btw,
+        taxScheme: taxSchemeIDVAT
+      })]
+    });
+    const accountingSupplierParty = new AccountingSupplierParty({
+      party: supplierParty
+    });
+
+    const customerLegalEntity = new PartyLegalEntity({
+      registrationName: savedInvoice.client.name,
+      companyID: savedInvoice.client.btw
+    });
+    const customerEndpointScheme = ENDPOINT_SCHEMES.find(scheme => scheme.country === savedInvoice.client.country);
+    const customerEndpointID = new UdtIdentifier(savedInvoice.client.btw, {
+      schemeID: customerEndpointScheme ? customerEndpointScheme.schemeID : ''
+    });
+    const customerParty = new Party({
+      EndpointID: customerEndpointID,
+      partyLegalEntities: [customerLegalEntity],
+      postalAddress: customerPostalAddress,
+      partyTaxSchemes: [new PartyTaxScheme({
+        companyID: savedInvoice.client.btw,
+        taxScheme: taxSchemeIDVAT
+      })]
+    });
+    const accountingCustomerParty = new AccountingCustomerParty({
+      party: customerParty
+    });
+
+    const legalMonetaryTotal = new LegalMonetaryTotal({
+      lineExtensionAmount: new UdtAmount(savedInvoice.money.totalWithoutTax.toFixed(2), currencyID),
+      taxInclusiveAmount: new UdtAmount(savedInvoice.money.total.toFixed(2), currencyID),
+      taxExclusiveAmount: new UdtAmount(savedInvoice.money.totalWithoutTax.toFixed(2), currencyID),
+      payableAmount: new UdtAmount(savedInvoice.money.total.toFixed(2), currencyID)
+    });
+
+
+    let taxObject: { id: string | UdtIdentifier; percent: string | UdtPercent; taxScheme: TaxScheme | undefined; taxExemptionReasonCode?: string | undefined };
+    let classifiedTaxObject: { id: string | UdtIdentifier; percent: string | UdtPercent; taxScheme: TaxScheme | undefined };
+    if (savedInvoice.client.country !== DEFAULT_COUNTRY_CODE) {
+      taxObject = {
+        id: 'AE',
+        percent: '0',
+        taxExemptionReasonCode: 'VATEX-EU-AE',
+        taxScheme: taxSchemeIDVAT
+      }
+      classifiedTaxObject = {
+        id: 'AE',
+        percent: '0',
+        taxScheme: taxSchemeIDVAT
+      }
+    } else {
+      taxObject = {
+        id: 'S',
+        percent: '21',
+        taxScheme: taxSchemeIDVAT
+      }
+      classifiedTaxObject = taxObject;
```

Comment:
Add some constants explaining what AE, S and VATEX-EU-AE is

Reply:
Done